src/utils/uiUtils.ts

```diff
@@ -31,27 +31,28 @@ export async function promptForOpenOutputChannel(message: string, type: DialogTy
     let result: vscode.MessageItem;
     switch (type) {
         case DialogType.info:
-            result = await vscode.window.showInformationMessage(message, DialogOption.OPEN, DialogOption.CANCEL);
+            result = await vscode.window.showInformationMessage(message, DialogOption.open, DialogOption.cancel);
             break;
         case DialogType.warning:
-            result = await vscode.window.showWarningMessage(message, DialogOption.OPEN, DialogOption.CANCEL);
+            result = await vscode.window.showWarningMessage(message, DialogOption.open, DialogOption.cancel);
             break;
         case DialogType.error:
-            result = await vscode.window.showErrorMessage(message, DialogOption.OPEN, DialogOption.CANCEL);
+            result = await vscode.window.showErrorMessage(message, DialogOption.open, DialogOption.cancel);
             break;
         default:
             break;
     }
 
-    if (result === DialogOption.OPEN) {
+    if (result === DialogOption.open) {
         terraformChannel.show();
     }
 }
 
 export namespace DialogOption {
-    export const OK: vscode.MessageItem = { title: "OK" };
-    export const CANCEL: vscode.MessageItem = { title: "Cancel", isCloseAffordance: true };
-    export const OPEN: vscode.MessageItem = { title: "Open" };
+    export const ok: vscode.MessageItem = { title: "OK" };
```

Comment:
i like the upper case here, is there any special reason to change it to lower case?

Reply:
Because I saw some other extensions are using the lower case:
- vscode-tomcat: https://github.com/adashen/vscode-tomcat/blob/develop/src/DialogMessage.ts
- vscode-azurefuntions: https://github.com/Microsoft/vscode-azuretools/blob/master/ui/src/DialogResponses.ts

Reply:
got that.